apps/sync/test/unit/sync-engine/kinto-mock.js

```diff
@@ -155,7 +156,7 @@ var Kinto = (function() {
   HttpCodeKintoCollectionMock.prototype = {
     sync() {
       var err = new Error();
-      err.request = {
+      err.response = {
```

Comment:
Corrected this as well, while I was at it - Kinto.js 1.0.0-rc4 throws an error that includes the `response` (not `request`) from the network request, when it encounters unexpected HTTP status codes during sync.